src/main/java/org/example/algorithms/CheckArraySortedNot.java

```diff
@@ -0,0 +1,32 @@
+public class  CheckArraySortedNot{
```

Comment:
Package info is missing at the top of the class.

```java
package org.example.algorithms;
```

See other classes for reference. Thanks.